tagmanager/ctags/ctags.c

```diff
@@ -49,6 +49,7 @@
 
 
 
+#ifndef GLIB_MAJOR_VERSION
```

Comment:
Are we ever going to exchange ctags.c with upstream?  I doubt it, it now has lots of changes.  So I guess it won't hurt to change it some more by deleting stuff.  But that should be its own PR.